#!/user/bin/env python

PACKAGE = "pcl_obstacle_detection"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Ground_Removal_rMin", double_t,    0, "Min Distance Detected", 0.1,  0, 100)
gen.add("Ground_Removal_rMax", double_t,    0, "Max Distance Dectected", 120,  0, 200)
gen.add("Ground_Removal_tHmin", double_t,    0, "Min Ground Height", -2,  -10, 10)
gen.add("Ground_Removal_tHmax", double_t,    0, "Max Ground Height", -0.5,  -10, 10)
gen.add("Ground_Removal_tHDiff", double_t,    0, "Diff in Height", 0.2,  -10, 10)
gen.add("Ground_Removal_hSensor", double_t,    0, "Sensor Height", 1.5,  -10, 10)

exit(gen.generate(PACKAGE, "pcl_obstacle_detection", "pcl_obstacle_detection_"))